modules/nodes-pods-pod-disruption-about.adoc

```diff
@@ -30,6 +30,8 @@ parts:
 
 [NOTE]
 ====
+`Available` refers to the number of pods that are passing the readiness probes.
```

Comment:
not all pods define readiness probes.  Is the actual definition pods that are declaring the `Ready=True` condition?

Reply:
Yes, that is correct. It should be defined as pods that have condition Ready=True

Reply:
As per the conversation, I will update the doc.

Reply:
Can we accurately state what's considered in determining when a pod is `Ready=True` ?

Reply:
@sdodson can you confirm the above comment? If that suffices I would make the changes accordingly.

Reply:
Sure that's fine.

Reply:
Great, I will make the changes accordingly.